<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Bhargavi" failures="0" tests="2" name="com.aventstack.extentreports.repoter.ExtentReportwithTestNg" time="0.196" errors="1" timestamp="2024-09-18T12:49:53 IST" skipped="0">
  <testcase classname="com.aventstack.extentreports.repoter.ExtentReportwithTestNg" name="testcase1" time="0.027"/>
  <testcase classname="com.aventstack.extentreports.repoter.ExtentReportwithTestNg" name="testcase2" time="0.169">
    <error message="The specified procedure could not be found" type="java.io.IOException">
      <![CDATA[java.io.IOException: The specified procedure could not be found
at java.base/java.io.WinNTFileSystem.canonicalize0(Native Method)
at java.base/java.io.WinNTFileSystem.canonicalize(WinNTFileSystem.java:465)
at java.base/java.io.File.getCanonicalPath(File.java:626)
at org.apache.commons.io.FileUtils.requireCanonicalPathsNotEquals(FileUtils.java:2702)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:801)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:834)
at org.apache.commons.io.FileUtils.copyFile(FileUtils.java:742)
at com.aventstack.extentreports.repoter.ExtentReportwithTestNg.Capturingthescreenshot(ExtentReportwithTestNg.java:74)
at com.aventstack.extentreports.repoter.ExtentReportwithTestNg.testcase2(ExtentReportwithTestNg.java:67)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </error>
  </testcase> <!-- testcase2 -->
</testsuite> <!-- com.aventstack.extentreports.repoter.ExtentReportwithTestNg -->
